frontend/src/features/modules/decls/VerbPanel.tsx

```diff
@@ -0,0 +1,67 @@
+import { useNavigate } from 'react-router-dom'
+import { Badge } from '../../../components/Badge'
+import type { Ref, Type, Verb } from '../../../protos/xyz/block/ftl/v1/schema/schema_pb'
+
+const DataRef = ({ heading, r }: { heading: string; r: Ref }) => {
```

Comment:
ubernit: we can use `ref` instead of `r`. It took me a sec to know what `r` was here 🤦

Reply:
Looks like `ref` is a reserved prop name in React because it gets handled differently and ends up being undefined whenever you reference it :(((